Tidy up of detail from Release Checklist (#3413)
adoptium/temurin-build: RELEASING.md

- Update [releaseVersions](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/regeneration/release_pipeline_generator.groovy#L6) with correct incoming release versions.
- Update https://github.com/adoptium/mirror-scripts/blob/master/releasePlan.cfg with expected tag, detail see https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes

#### Branching message for build related repositories

> Post the below  message to the #build & #release channels in Slack:

Reviewer: ```suggestion
Post the below message to the #build & #release channels in Slack:
```

> In Preparation for next weeks release, I'm proposing to branch the

Reviewer 2: Perhaps have this as a single line in backticks to make it clearer which bit to cut & paste?

following repositories, in order that this branch becomes the baseline
for the release. Shout now if you need to get any PRs merged for the release,
as today is the last day, the following repositories will be branched:

>   ( temurin-build, ci-jenkins-pipelines, jenkins-helper).

Reviewer: ```suggestion
temurin-build, ci-jenkins-pipelines, and jenkins-helper.
```

#### Create release branch on below repositories:
 
- temurin-build <https://github.com/adoptium/temurin-build>
- ci-jenkins-pipelines <https://github.com/adoptium/ci-jenkins-pipelines>
- jenkins-helper <https://github.com/adoptium/jenkins-helper>

> These branches should be named according to the following format (vYYYY.MM.NN) ,e.g v2023.03.01 , whereby the final element is an incremental counter appended to the year and month of the release.

Reviewer: The v-string looks a lot like a date. Could we change the last full-stop to a plus or a dash or something, to prevent misunderstandings? E.g. v2023.03+01 or v2023.03-01

Reviewer: ```suggestion
These branches should be named according to the following format (vYYYY.MM+NN) ,e.g v2023.03+01 , whereby the final element is an incremental counter appended to the year and month of the release.
```

#### Code Freeze message

Paste the below message into the #release channel in Slack:

> With under a week to go until releases, we are entering a lockdown period for the following repositories: <a href='https://github.com/adoptium/temurin-build/pulls'>temurin-build</a>, <a href='https://github.com/adoptium/ci-jenkins-pipelines/pulls'>ci-jenkins-pipelines</a>, <a href='https://github.com/adoptium/github-release-scripts/pulls'>github-release-scripts</a>, <a href='https://github.com/adoptium/containers/pulls'>containers</a>, <a href='https://github.com/adoptium/installer/pulls'>installer</a>, <a href='https://github.com/adoptium/mirror-scripts'>mirror-scripts</a>.

Reviewer: I'm not sure I understand why these links are in html. Slack doesn't appear to parse html when I paste it into there, at least not in the desktop client.

Perhaps plan text is enough?

Reviewer: ```suggestion
With under a week to go until releases, we are entering a lockdown period for the following repositories: temurin-build, ci-jenkins-pipelines, github-release-scripts, containers, installer, and mirror-scripts.
```

If you need to submit a pr for any of these repos during this period, you should:

>  - Add a comment saying “Approval to merge during the lockdown cycle please” and post in the appropriate slack channel for awareness. This can be done before the PR is finalised

Reviewer: ```suggestion
 - Add a comment saying “Approval to merge during the lockdown cycle please” and post in the appropriate slack channel for awareness. This can be done before the PR is finalised.
```

>  - Add a note into this channel saying you are requesting the approval with a link to the comment in the first bullet point

Reviewer: ```suggestion
 - Add a note into this channel saying you are requesting the approval with a link to the comment in the first bullet point.
```

>  - The comment should have approval from at least one build committer and one PMC member to indicate that they agree it is critical that it goes in

Reviewer: ```suggestion
 - The comment should have approval from at least one build committer and one PMC member to indicate that they agree it is critical that it goes in.
```

>  - The PR can be merged after 2 hours of the post going into the build channel (to give people time to object ... This delay may be skipped in the case where the delay will result in something breaking within that time.

Reviewer: ```suggestion
 - The PR can be merged after 2 hours of the post going into the build channel (to give people time to object). This delay may be skipped in cases where the delay will result in something breaking within that time.
```

Only include "critical" fixes (i.e. those which will otherwise cause a build break or other problem which will prevent shipping the release builds).
This stops last minute changes going in, which may destabilise things. "installer" repo might have exception due to the fact it requires new version of build.
If a change has to go in during this "lockdown" period it should be done by posting a comment saying "Requesting approval to merge during the lockdown period. Please thumbs up the comment to approve" in Slack release channel.
If two committers into the repository express approval then the change can be merged during the lockdown period.

> #### Finally lockdown main branches of below repositories

Reviewer 2: Can we clarify what "lockdown main branches" means?

- temurin-build <https://github.com/adoptium/temurin-build>
- ci-jenkins-pipelines <https://github.com/adoptium/ci-jenkins-pipelines>
- jenkins-helper <https://github.com/adoptium/jenkins-helper>
- github-release-scripts <https://github.com/adoptium/github-release-scripts>
- containers <https://github.com/adoptium/containers>
- installer <https://github.com/adoptium/installer>
- mirror-script <https://github.com/adoptium/mirror-scripts>

> #### Release pipelines and jobs need to be re-generated with new tags by Release Champion:

Reviewer: ```suggestion
#### Release pipelines and jobs need to be re-generated with new tags by the Release Champion:
```
